assets/src/block-templates/side-image-with-text-and-cta/block.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 2,
+  "name": "planet4-block-templates/side-image-with-text-and-cta",
+  "title": "Side image with text and CTA",
```

Comment:
Probably not, if I remember correctly we previously agreed that since in the Handbook we use the English names for the blocks/patterns/etc, we shouldn't translate them in the backend 🤔 Otherwise it might be confusing for editors

Reply:
I agree to don't translate `name`, but what about the `title`?

Reply:
Yeah, I think because we use the English titles in the Handbook it would be confusing to have them translated in the editor. I think @sagarsdeshmukh worked on this?

Reply:
Yes, it was my proposal to keep the block name/title, style name etc in base language, to minimise naming confusion in different languages while searching them in handbook for more details. Also the [native Gutenberg block](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/button/block.json#L5) names/style names are not offer translation.

Reply:
I did not implement it in this PR, but Gutenberg (and WP) has a system to translate the metadata in `block.json`, using the `textdomain` included in the metadata, this function [translateBlockSettingUsingI18nSchema](https://github.com/WordPress/gutenberg/blob/ec588b48861ac86a8f96589d3da3c50cacb406e5/packages/blocks/src/api/registration.js#L306-L356) during metadata reading (the same exists server side if I remember), and a file [describing the translation schema (fields to translate)](https://github.com/WordPress/gutenberg/blob/5d2e14a5d475fc62a2b343481dd65bc2dc5d4cad/packages/blocks/src/api/i18n-block.json) for blocks.